k9mail/src/main/java/com/fsck/k9/activity/MessageLoaderHelper.java

```diff
@@ -416,7 +421,7 @@ public void loadMessageRemoteFailed(Account account, String folder, String uid,
         void onMessageDataLoadFailed();
 
         void onMessageViewInfoLoadFinished(LocalMessage localMessage, MessageViewInfo messageViewInfo);
-        void onMessageViewInfoLoadFailed(LocalMessage localMessage);
+        void onMessageViewInfoLoadFailed(LocalMessage localMessage, MessageViewInfo messageViewInfo);
```

Comment:
now that you mention it, now that #1501 is fixed we can get rid of the LocalMessage parameter in the success case as well :)